great_expectations/rule_based_profiler/types/data_assistant_result/volume_data_assistant_result.py

```diff
@@ -28,111 +30,248 @@ def plot(
         Args:
             prescriptive: Type of plot to generate, prescriptive if True, descriptive if False
             theme: Altair top-level chart configuration dictionary
+            include_column_names: A list of columns to chart
+            exclude_column_names: A list of columns to avoid charting
         """
-        # TODO: <Alex>ALEX Currently, only one Domain key (with domain_type of MetricDomainTypes.TABLE) is utilized; enhancements may require additional Domain key(s) with different domain_type value(s) to be incorporated.</Alex>
-        # noinspection PyTypeChecker
-        attributed_metrics_by_domain: Dict[Domain, Dict[str, ParameterNode]] = dict(
+        assert not (
+            include_column_names is not None and exclude_column_names is not None
+        ), "You may either use `include_column_names` or `exclude_column_names` (but not both)."
```

Comment:
Isn't it preferred to raise a `ValueError` here?

Reply:
yup I like that better - updated conditional